whitelist_python.py

```diff
@@ -0,0 +1,3 @@
+import collections
+
+collections.defaultdict(list).default_factory
```

Comment:
Consider the example code E = `defaultdict(list).default_factory = None`. What we want is a whitelist file W such that vulture makes the following report:

vulture(E + W) -> no dead code

It is ok to have the following (since it's the reason we need a whitelist file to begin with):

vulture(E) -> dead code

We always need to make sure that whitelist files don't contain unused code, but this is the case in my proposal. 

vulture(W) -> no dead code

We should add a third test for the last case.